Makefile

```diff
@@ -39,6 +39,7 @@ EXEEXT := $(shell $(PYTHON) -c \
 NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
+NPM ?= $(PWD)/deps/npm/bin/npm-cli.js
```

Comment:
@bnoordhuis Thanks for the hint! I’ve went with quoting the uses below, if that’s okay, that makes it a bit less messy there…